### PR TITLE
[ui][fix] fill connection credentials in lower case by data-testid

### PR DESCRIPTION
**Pull request by mcada**
//test: `@validate-connection-credentials`

### CODE REVIEW

**Comment by syndesisqe** [comment]
Following scenarios failed:
```
Integration - Database | Smoke - Periodic invocation to Insert
Connection - CRUD | Credentials | <font color="red"><b>FAILED</b></font> | mcada@redhat.com
Connection - CRUD | Create and delete | <font color="red"><b>FAILED</b></font> | mcada@redhat.com
Connection - CRUD | Kebab menu | <font color="red"><b>FAILED</b></font> | mcada@redhat.com
Connection - CRUD | Kebab menu edit and view
```

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

File: ui-tests/src/test/java/io/syndesis/qe/steps/connections/wizard/phases/ConfigureConnectionSteps.java [hunk 1]
```diff
@@ -1,58 +1,46 @@
...
     }
 
-    private void callCorrectFormFill(Map<String, String> properties, boolean isEditPage) {
-        if (isEditPage) {
-            new Form($(By.className("integration-editor-layout__body")).shouldBe(visible)).fillById(properties);
-        } else {
-            configureConnectionPage.fillForm(properties);
-        }
+    private void callCorrectFormFill(Map<String, String> properties) {
+        new Form($(By.className("integration-editor-layout__body")).shouldBe(visible)).fillByTestId(properties.entrySet().stream()
```
Comment (asmigala) [inline ui-tests/src/test/java/io/syndesis/qe/steps/connections/wizard/phases/ConfigureConnectionSteps.java]: Could you please refactor a bit to make more readable? Or maybe just change the formatting.

File: ui-tests/src/test/java/io/syndesis/qe/steps/connections/wizard/phases/ConfigureConnectionSteps.java [hunk 2]
```diff
@@ -1,58 +1,46 @@
...
         }
     }
 
-    private void callCorrectFormFill(Map<String, String> properties, boolean isEditPage) {
-        if (isEditPage) {
-            new Form($(By.className("integration-editor-layout__body")).shouldBe(visible)).fillById(properties);
-        } else {
-            configureConnectionPage.fillForm(properties);
-        }
+    private void callCorrectFormFill(Map<String, String> properties) {
```
Comment (asmigala) [inline ui-tests/src/test/java/io/syndesis/qe/steps/connections/wizard/phases/ConfigureConnectionSteps.java]: Please rename, it now always calls the same form

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by avano** [comment]
retest this please

**Comment by syndesisqe** [comment]
Following scenarios failed:
```
Integration - Database | Smoke - Periodic invocation to Insert
Connection - CRUD | Credentials | <font color="red"><b>FAILED</b></font> | mcada@redhat.com
Connection - CRUD | Create and delete | <font color="red"><b>FAILED</b></font> | mcada@redhat.com
Connection - CRUD | Kebab menu | <font color="red"><b>FAILED</b></font> | mcada@redhat.com
Connection - CRUD | Kebab menu edit and view
```

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by mcada** [comment]
@avano please merge this PR, the tests wont all pass due to https://github.com/syndesisio/syndesis/issues/5582 but I will fix them in separate PR as it is a different issue

**Comment by syndesisqe** [comment]
Following scenarios failed:
```
Connection - CRUD | Credentials | <font color="red"><b>FAILED</b></font> | mcada@redhat.com
```

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Pull request merged**